website/e2e/entrypoint.sh

```
@@ -0,0 +1,2 @@
+#!/bin/bash
+/wait-for-it.sh apostrophe:3000 -t 60 -- sh -c "wget -qO- http://apostrophe:3000/ > /dev/null && npx playwright test \"$@\"" 
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)